serve/graph/model/models_gen.go

```diff
@@ -3,19 +3,113 @@
-	FromHeight *int `json:"from_height,omitempty"`
-	// Maximum block height up to which Blocks should be fetched, exclusive. If unspecified, there is no upper bound.
-	ToHeight *int `json:"to_height,omitempty"`
-	// Minimum timestamp from which to start fetching Blocks, inclusive. Blocks created at or after this time will be included.
-	FromTime *time.Time `json:"from_time,omitempty"`
-	// Maximum timestamp up to which to fetch Blocks, exclusive. Only Blocks created before this time are included.
-	ToTime *time.Time `json:"to_time,omitempty"`
+type MessageValue interface {
+	IsMessageValue()
+}
+
+type BankMsgSend struct {
+	FromAddress string `json:"from_address"`
+	ToAddress   string `json:"to_address"`
+	Amount      string `json:"amount"`
+}
+
+func (BankMsgSend) IsMessageValue() {}
+
+// Input parameters required when the message type is `send`.
+type BankMsgSendInput struct {
+	// Filter by `from_address`.
+	FromAddress *string `json:"from_address,omitempty"`
+	// Filter by `to_address`.
+	ToAddress *string `json:"to_address,omitempty"`
+	// Filter by `amount`.
+	Amount *string `json:"amount,omitempty"`
+}
+
+type MemFile struct {
+	Name string `json:"Name"`
+	Body string `json:"Body"`
+}
+
+type MemFileInput struct {
+	Name *string `json:"Name,omitempty"`
+	Body *string `json:"Body,omitempty"`
+}
+
+type MemPackage struct {
+	Name  string     `json:"Name"`
+	Path  string     `json:"Path"`
+	Files []*MemFile `json:"Files,omitempty"`
+}
+
+type MemPackageInput struct {
+	Name  *string         `json:"Name,omitempty"`
+	Path  *string         `json:"Path,omitempty"`
+	Files []*MemFileInput `json:"Files,omitempty"`
+}
+
+type MsgAddPackage struct {
+	Creator string      `json:"creator"`
+	Package *MemPackage `json:"package"`
+	Deposit string      `json:"deposit"`
+}
+
+func (MsgAddPackage) IsMessageValue() {}
+
+// Input parameters required when the message type is `add_package`.
+type MsgAddPackageInput struct {
+	// Filter by `creator`.
+	Creator *string `json:"creator,omitempty"`
+	// Filter by `package`.
+	Package *MemPackageInput `json:"package,omitempty"`
+	// Filter by `deposit`.
+	Deposit *string `json:"deposit,omitempty"`
+}
+
+type MsgCall struct {
+	Caller  string   `json:"caller"`
+	Send    string   `json:"send"`
+	PkgPath string   `json:"pkg_path"`
+	Func    string   `json:"func"`
+	Args    []string `json:"args,omitempty"`
+}
+
+func (MsgCall) IsMessageValue() {}
+
+// Input parameters required when the message type is `exec`.
+type MsgCallInput struct {
+	// Filter by `caller`.
+	Caller *string `json:"caller,omitempty"`
+	// Filter by `send`.
+	Send *string `json:"send,omitempty"`
+	// Filter by `pkg_path`.
+	PkgPath *string `json:"pkg_path,omitempty"`
+	// Filter by `func`.
+	Func *string `json:"func,omitempty"`
+	// Filter by `args`, Arguments are checked in the order of the argument array, and arguments that are not checked are left blank.
+	Args []string `json:"args,omitempty"`
+}
```

Comment:
The same as previous comments. These Go docs will be shown to the user as API documentation. Would be great if you could give more info.

Reply:
I added comments to the graphql schema.